src/lib/components/Button/README.md

```diff
@@ -275,126 +258,126 @@ state are automatically disabled to prevent unwanted interaction.
 
 Filled buttons in feedback state:
 
-<Playground>
+```docoff-react-preview
   <Button
     label="Success"
     color="success"
-    feedbackIcon={<Icon icon="success" />}
+    feedbackIcon={<rui-icon icon="success" />}
   />
   <Button
     label="Success"
     labelVisibility="none"
     color="success"
-    feedbackIcon={<Icon icon="success" />}
-    endCorner={<Badge color="danger" label={3} />}
+    feedbackIcon={<rui-icon icon="success" />}
+    endCorner={<Badge color="danger" label={'3'} />}
   />
-  <Placeholder dark inline>
+  <docoff-placeholder dark inline>
     <Button
       label="Light"
       color="light"
-      feedbackIcon={<Icon icon="success" />}
+      feedbackIcon={<rui-icon icon="success" />}
     />
-  </Placeholder>
+  </docoff-placeholder>
   <Button
     label="Dark"
     color="dark"
-    feedbackIcon={<Icon icon="success" />}
+    feedbackIcon={<rui-icon icon="success" />}
   />
   <Button
     label="Primary"
     feedbackIcon={(
       <span className="d-inline-flex animation-spin-counterclockwise">
-        <Icon icon="loading" />
+        <rui-icon icon="loading" />
       </span>
     )}
   />
-</Playground>
+```
 
 Outline buttons in feedback state:
 
-<Playground>
+```docoff-react-preview
   <Button
     label="Success"
     priority="outline"
     color="success"
-    feedbackIcon={<Icon icon="success" />}
+    feedbackIcon={<rui-icon icon="success" />}
   />
   <Button
     label="Success"
     labelVisibility="none"
     priority="outline"
     color="success"
-    feedbackIcon={<Icon icon="success" />}
-    endCorner={<Badge color="danger" label={3} />}
+    feedbackIcon={<rui-icon icon="success" />}
+    endCorner={<Badge color="danger" label={'3'} />}
   />
-  <Placeholder dark inline>
+  <docoff-placeholder dark inline>
     <Button
       label="Light"
       priority="outline"
       color="light"
-      feedbackIcon={<Icon icon="success" />}
+      feedbackIcon={<rui-icon icon="success" />}
     />
-  </Placeholder>
+  </docoff-placeholder>
   <Button
     label="Dark"
     priority="outline"
     color="dark"
-    feedbackIcon={<Icon icon="success" />}
+    feedbackIcon={<rui-icon icon="success" />}
   />
   <Button
     label="Primary"
     priority="outline"
     feedbackIcon={(
       <span className="d-inline-flex animation-spin-counterclockwise">
-        <Icon icon="loading" />
+        <rui-icon icon="loading" />
```

Comment:
Animation is somehow broken, it does not circle in the middle of the button, but around the center of the button,

![image](https://user-images.githubusercontent.com/10213870/209100484-8d0bc860-8b8b-4cf2-965e-e05af6b201b2.png)

Reply:
This is the same problem as in `animation.md`. Lets resolve it there.